app/models/court_application_defendant_summary.rb

```diff
@@ -25,7 +25,7 @@ def to_builder
       end
       defendant_summary.dateOfNextHearing date_of_next_hearing
       defendant_summary.proceedingsConcluded proceedings_concluded?
-      defendant_summary.masterDefendantId defendant.masterDefendantId
+      defendant_summary.masterDefendantId defendant_id
```

Comment:
I'm not sure this is a good idea. In production, I don't think we can always assume that masterDefendantId is going to be the same as subjectId - for prosecution cases sometimes it's different.

Reply:
No it wont be the same in CP they have another table for subjectSummary I think until we fully verify this we should not block us from implementing the masterDefendantId as that will be be defendantId as confirmed by them today, I will add a comment if that helps, for now so we can change the subjectId to implement the same as them that needs another ticket this is just so we can implement the ACD-522 match the correct defendant to court application

Reply:
Sorry, I wrote "subjectId" when I meant "defendantId". If they've confirmed that subjectSummary->masterDefendantId will always be the same as prosecutionCase->defendantId then that's awesome news and saves us some very messy code!